connector/dao/src/main/java/it/pagopa/selfcare/mscore/connector/dao/DelegationConnectorImpl.java

```diff
@@ -153,4 +154,25 @@ public Delegation findAndActivate(String from, String to, String productId) {
         FindAndModifyOptions findAndModifyOptions = FindAndModifyOptions.options().upsert(false).returnNew(true);
         return delegationMapper.convertToDelegation(repository.findAndModify(query, update, findAndModifyOptions, DelegationEntity.class));
     }
+
+    @Override
+    public void updateDelegation(Institution institutionUpdate) {
+
+        if (institutionUpdate.isDelegation()) {
```

Comment:
you could add some comments about this if....for ex. to explain when isDelegation has value to true